src/middlewared/middlewared/utils/__init__.py

```diff
@@ -254,6 +255,11 @@ def sw_version():
     return VERSION
 
 
+@functools.cache
+def manifest_version():
```

Comment:
Please check the various system plugins. I believe we already have a method that returns the information you want. If there isn't, please rename this function to what it's actually doing. It's reading the manifest file to determine the scale release, not return the manifest file version?

Reply:
We don't unfortunately have that method already - we have a method which returns the verbose name or Full Name as we call it.
I'll rename the method to reflect reality.

Reply:
With https://github.com/truenas/middleware/commit/e42cd6e1da74b1bace5aa83a68db9b6dd8b02c31 merged, you can call `system.version False` so you don't have to add this helper function anymore.